applications/ananke/src/ananke_callback_sup.erl

```diff
@@ -1,88 +0,0 @@
-%%%-------------------------------------------------------------------
```

Comment:
@icehess is this the task sup below now?  the callback feature still works correct?